frontend/components/header/HeaderAppPage.vue

```diff
@@ -48,13 +52,15 @@
 </template>
 
 <script setup lang="ts">
+import type { DiscussionText } from "~/types/card-discussion-text";
```

Comment:
We're not using this, so it's not needed to import it, right, @UnknownSean8? Maybe something from an earlier way you were working on this?